src/Symfony/Component/Config/Definition/BaseNode.php

```diff
@@ -45,15 +44,13 @@ abstract class BaseNode implements NodeInterface
      * @throws \InvalidArgumentException if the name contains a period
      */
     public function __construct(
-        ?string $name,
+        protected ?string $name,
```

Comment:
This changes the property type from `string` to `?string`.
That's a BC break and I don't think we want this.